.uberenv_config.json

```diff
@@ -3,10 +3,11 @@
 "package_version" : "develop",
 "package_final_phase" : "hostconfig",
 "package_source_dir" : "../..",
-"spack_url": "https://github.com/davidbeckingsale/spack",
-"spack_branch": "feature/allow-untested-cuda-versions",
-"spack_commit": "46b22d0f6227f6b12bab712bda5b916a53cfc67d",
+"spack_url": "https://github.com/spack/spack.git",
+"spack_commit": "c2af154cd24ab3ef501a7efc020f818ef60aa552",
```

Comment:
When I was trying to figure out how to update spack/uberenv for Umpire a while back, I had no idea how certain commits in this config file were determined and why one specific commit would work and others would cause crashes. Can we document why this specific commit was picked somewhere? And if I wanted to update this in the future, I'd like to know where to start in finding an updated commit.

Reply:
I just followed what RAJA is doing so it's consistent. @adrienbernede picked the specific commit.

Reply:
@kab163 I can understand your frustration. I tried to summarize the logic in a how-to article of radiuss-shared-ci.

Warning: the length is proportional to the amount of magic in the choice, it’s not exactly a science :)

https://radiuss-shared-ci.readthedocs.io/en/latest/sphinx/user_guide/how_to.html#choose-a-spack-reference-commit-or-branch

Feel free to ask questions: they will motivate me to create more articles.